node/src/tests.rs

```diff
@@ -194,7 +194,7 @@ fn export_current_state() {
 
 		// Let it produce some blocks.
 		// This fails if is not a minimum of 25
-		thread::sleep(Duration::from_secs(25));
+		thread::sleep(Duration::from_secs(35));
```

Comment:
why ?

Reply:
I guess because of the JIT compilation issue. but the CI was complaining:(

Reply:
This can [hopefully] be reverted once 993 is merged.

Reply:
indeed